README.md

```diff
@@ -44,7 +44,7 @@ We are excited about your interest in CORTX and hope you will join us. We take o
 
 ## Get Started
 
-- Download a [CORTX virtual machine image](doc/ova/1.0.4/CORTX_on_Open_Virtual_Appliance.rst) to get up and running in minutes.
+- Download a [CORTX virtual machine image](doc/ova/2.0.0/CORTX_on_Open_Virtual_Appliance.rst) to get up and running in minutes.
```

Comment:
@hessio, link for 2.0.0 has changed to https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/CORTX_on_Open_Virtual_Appliance_PI-2.rst so correct the same

Reply:
Hi @mukul-seagate11, I've updated the link, wouldn't it be better if the latest link always stays the same instead?
Refer to [discussion](https://github.com/Seagate/cortx/discussions/1185)